File: inventory-api/src/main/java/com/kruger/ec/dto/core/KgrtVaccinationStatusDto.java
package com.kruger.ec.dto.core;

import com.kruger.ec.dto.AbstractDto;
import lombok.Getter;
import lombok.Setter;

import java.util.Date;
import java.util.List;

@Getter
@Setter
public class KgrtVaccinationStatusDto extends AbstractDto<Long> {

    private Long vaccinationStatuslId;
    
    private String vaccinationStatusvName;
    
    private String vaccinationStatusvObservation;
    
    private Boolean vaccinationStatusbActive;
    
    private Date audidCreationDate;
    
    private String audivUserCreation;
    
    private Date audidModificationDate;
    
    private String audivUserModification;
    
    private Date audidDeleteDate;
    
    private String audivUserDelete;
    private KgrtVaccinationDto vaccinationlIdDTO;

}

File: inventory-ui/src/app/dto/KgrtVaccinationStatusDto.ts
import {KgrtVaccinationDto} from "@/dto/KgrtVaccinationDto";

export class KgrtVaccinationStatusDto {
  vaccinationStatuslId: number;
  vaccinationStatusvName: string;
  vaccinationStatusvObservation: string;
  vaccinationStatusbActive: boolean;
  audidCreationDate: Date;
  audivUserCreation: string;
  audidModificationDate: Date;
  audivUserModification: string;
  audidDeleteDate: Date;
  audivUserDelete: string;
  vaccinationlIdDTO: KgrtVaccinationDto;
}
